src/reason_pprint_ast.ml

```diff
@@ -3014,6 +3011,23 @@ class printer  ()= object(self:'self)
         in
         let rightItm = self#simplifyUnparseExpr rightExpr in
         Some (label ~space:forceSpace (atom prefixStr) rightItm)
+      | (Infix s, [(_, e1); (_, e2)]) when s.[0] = '#' ->
```

Comment:
I think it's wrong that this is considered `Infix`.  Infix function application cannot be passed as an argument without guarding parens `add (3 + 2) (4 + 5)`. But I believe `##` can be passed as an argument to a function. `add x##y z##q`